src/db/models/organization.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+import {sequelize} from './../mysql-connector';
+import * as Sequelize from 'sequelize';
+
+export const Organization = sequelize.define('organizations', {
+    id: {
+        type: Sequelize.BIGINT,
+        primaryKey: true,
+        autoIncrement: true,
+    },
+    name: {
+        type: Sequelize.STRING,
+    },
+    phone: {
+        type: Sequelize.STRING,
+    },
+    email: {
+        type: Sequelize.STRING,
+    },
+    owner_id: {
+        type: Sequelize.STRING,
+    },
+    aws_account_status: {
+        type: Sequelize.STRING,
+    },
+    aws_client_id: {
+        type: Sequelize.STRING,
+    },
+    aws_secret_key: {
+        type: Sequelize.STRING,
+    },
+    aws_account_key: {
+        type: Sequelize.STRING,
+    },
+    createdAt: {
```

Comment:
Why have you used two different naming conventions here? any specific reason for that?

Reply:
yes, Sequelize by default gets it as createdAt, so if i try to put it as created_at it will throw an error when i inset value. What i have to do is, make the field name as createdAt in the model and map it to created_at field name in the DB.